libs/application/templates/parental-leave/src/fields/Review/ReadOnlyReview.tsx

```diff
@@ -0,0 +1,337 @@
+                          getValueViaPath(
+                            application.answers,
+                            'otherParentName',
+                          ) as string[]
+                        }
+                      </Text>
+                    </GridColumn>
+                    <GridColumn span="6/12">
+                      <Text>
+                        {
+                          getValueViaPath(
+                            application.answers,
+                            'otherParentId',
+                          ) as string[]
+                        }
+                      </Text>
+                    </GridColumn>
+                  </GridRow>
+                </>
+              )}
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem
+            id="id_3"
+            label={formatMessage(m.paymentInformationSubSection)}
+            startExpanded={allItemsExpanded}
+          >
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="6/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'payments.bank',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+                <GridColumn span="6/12">
+                  {
+                    //TODO add the personal allowance questions when finished
+                  }
+                </GridColumn>
+              </GridRow>
+              <Box marginTop={3} />
+              <GridRow>
+                <GridColumn span="6/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'payments.pensionFund',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+                <GridColumn span="6/12">
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'payments.union',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+              </GridRow>
+              <Box marginTop={3} />
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Box marginTop={1} marginBottom={2}>
+                    <Text variant="h5">
+                      {formatMessage(m.privatePensionFundName)}
+                    </Text>
+                  </Box>
+                  <Text>
+                    {
+                      getValueViaPath(
+                        application.answers,
+                        'usePrivatePensionFund',
+                      ) as string[]
+                    }
+                  </Text>
+                </GridColumn>
+              </GridRow>
```

Comment:
Isn't grid row and column unnecessary here? Or are we looking for consistency in the markup? Maybe I'm overlooking something :)
```suggestion
              <Box marginTop={3}>
                  <Box marginTop={1} marginBottom={2}>
                    <Text variant="h5">
                      {formatMessage(m.privatePensionFundName)}
                    </Text>
                  </Box>
                  <Text>
                    {
                      getValueViaPath(
                        application.answers,
                        'usePrivatePensionFund',
                      ) as string[]
                    }
                  </Text>
              </Box>
```

Reply:
Thanks for these tips! Yeah was being consistent but I see all the unnecessary mark-up now. Should be cleaned up in the latest.